src/test/miner_tests.cpp

```diff
@@ -58,36 +60,43 @@ constexpr static struct {
     uint8_t extranonce;
     uint32_t nonce;
 } blockinfo[] = {
-    {2, 0x97a370c4}, {3,  0x2661ae42}, {2, 0x4a197044}, {4,  0x8ed93a46},
-    {2, 0xfb8032c2}, {2,  0xb254623e}, {2, 0x3792c05b}, {3,  0x329b0f7d},
-    {3, 0x1fdcd0f6}, {13, 0x68a8acb9}, {2, 0x486aa328}, {7,  0x54b615a2},
-    {6, 0x8b6c5636}, {10, 0x44d4159b}, {7, 0xab718100}, {4,  0xac18621f},
-    {5, 0xb3d6b4f7}, {3,  0x15486857}, {2, 0xff5e5b2a}, {6,  0x6ae15f86},
-    {4, 0x4cfaa426}, {2,  0xa32e2ab2}, {2, 0x68fce1de}, {2,  0x8553c3fe},
-    {2, 0x72c10730}, {2,  0x0fdae050}, {2, 0x6602c7a0}, {3,  0x2fdde145},
-    {5, 0x0ed2e715}, {7,  0x3953e5f8}, {3, 0x7d21a613}, {2,  0xb2725170},
-    {3, 0xa4c921cf}, {2,  0x0aa87056}, {2, 0xd1508092}, {4,  0x53c137b2},
-    {2, 0xa5bfb006}, {2,  0xdd200659}, {3, 0x5ce45789}, {2,  0xd57ca754},
-    {2, 0xabdf406e}, {4,  0xbcefc309}, {5, 0xccd062b9}, {2,  0xa4db21c3},
-    {3, 0xb673a715}, {3,  0x205e4702}, {3, 0xcec2859e}, {2,  0xbc2ba701},
-    {2, 0xd6837046}, {2,  0x4a0593ef}, {2, 0x5eeeb283}, {13, 0x22397a97},
-    {2, 0x31cc6463}, {3,  0xb3fc0716}, {4, 0x3299d7c1}, {2,  0x3a6d0c51},
-    {2, 0x9edd6d17}, {2,  0x6dce7a55}, {2, 0xe70c9925}, {3,  0x6b874479},
-    {7, 0x071241ca}, {4,  0xf7f26707}, {2, 0xc29ef5ae}, {6,  0x5c3d21b7},
-    {3, 0x3bca753c}, {3,  0x73a543c4}, {5, 0xb3384572}, {3,  0xbb78de4e},
-    {7, 0x2119c7c6}, {5,  0xdaa851e3}, {3, 0x279fe081}, {4,  0x6e894353},
-    {2, 0xa3b57648}, {2,  0xf53816bb}, {3, 0xa785a43f}, {3,  0xa4a813ab},
-    {4, 0xdee3b283}, {4,  0x9e0eef78}, {4, 0x0f87df2f}, {4,  0x8b76a010},
-    {2, 0xefc5c3e4}, {2,  0x32662ae3}, {2, 0xa8f47138}, {2,  0xc550b341},
-    {3, 0xa550cb97}, {2,  0x55955315}, {2, 0x4c3f3038}, {4,  0x1658a340},
-    {7, 0x29f9333b}, {3,  0xdf97ae03}, {5, 0x321aa10b}, {2,  0x9f3c21cf},
-    {3, 0x3568a31c}, {4,  0xceef8301}, {3, 0x09645c63}, {4,  0xb328d56f},
-    {4, 0xc4de303b}, {2,  0x50f80a5b}, {3, 0x1455e14c}, {4,  0x3da25948},
-    {2, 0x1fbb494e}, {8,  0xa41ad1b4}, {2, 0xee911aa6}, {2,  0xaea15a0c},
-    {8, 0xbcc24752}, {3,  0x41aa8978}, {2, 0x41795716}, {6,  0x7d6e019},
-    {3, 0xeb30b104}, {2,  0x88ba4d3c},
+    {1, 0xa9d9c5fe}, {2, 0xc4b8f377}, {3, 0x8cec161a}, {2, 0xc8925628},
```

Comment:
Joy. It seems like this will keep mutating. Is there a way to avoid this?

Reply:
No. Other than frying my machine, it's not a problem though, is it? Keeps the commits consistent.

Reply:
I'm just trying to understand the tests. Bitcoind is ~300K LOC, and I have only messed with fees, mempool, and mining. I have ~0 experience with validation code.